packages/plugins/integration-plane-ui/src/lib/components/api-key-dialog/api-key-dialog.component.html

```
@@ -0,0 +1,77 @@
+<nb-card class="api-key-dialog">
+	<nb-card-header class="d-flex align-items-center justify-content-between">
+		<h6 class="mb-0">{{ 'INTEGRATIONS.PLANE_PAGE.API_KEY_DIALOG.TITLE' | translate }}</h6>
+		<button nbButton ghost size="small" (click)="close()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)